packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -1420,7 +1438,7 @@ test.serial('time jumps forward', async t => {
   const schedules = await driver.getSchedule();
   t.is(schedules.nextAuctionSchedule?.startTime.absValue, 1570n);
   t.is(schedules.liveAuctionSchedule?.startTime.absValue, 1530n);
-  t.is(schedules.liveAuctionSchedule?.endTime.absValue, 1550n);
+  t.is(schedules.liveAuctionSchedule?.endTime.absValue, 1545n);
```

Comment:
why did this value change?

Reply:
See lines 1208 and 61 above.

Previously, `test('multiple collaterals'` was side-effecting the global constant `defaultParams`, changing the expected `lowestRate` for all following tests. Now that `defaultParams` is unchanging, these are the correct times.

Reply:
ah. double plus good hardening.